tock/tock/settings/dev.py

```diff
@@ -1,28 +1,49 @@
-from .base import *  # noqa
-
 import dj_database_url
 
+from .base import *  # noqa
+
 DEBUG = True
-TEMPLATE_DEBUG = True
+TEMPLATE_DEBUG = DEBUG
+DEBUG_TOOLBAR_PATCH_SETTINGS = False
 
-INTERNAL_IPS = ('127.0.0.1',)
+INTERNAL_IPS = [
+    '127.0.0.1',
+    '::1',
+]
```

Comment:
I'm curious what this will be used for?  For example, I often access my machine from a cell phone on my wifi network to ensure that mobile works OK, but then I'm not accessing it directly via the loopback interface and I might not have access to some functionality.

Is `::1` to support IPv6?